src/overlays/actors/ovl_Obj_Y2shutter/z_obj_y2shutter.c

```diff
@@ -28,24 +41,136 @@ const ActorInit Obj_Y2shutter_InitVars = {
     (ActorFunc)ObjY2shutter_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B9AEB0[] = {
+typedef enum {
+    /* 0x0 */ BARRED_SHUTTER,
+    /* 0x1 */ GRATED_SHUTTER,
+} ShutterType;
```

Comment:
Other files might want to use this (if they spawn one of these, for example), so it should go in the header.